community/kernel/src/main/java/org/neo4j/kernel/impl/storageengine/impl/recordstorage/StoreStatement.java

```diff
@@ -331,123 +338,119 @@ public Properties properties()
         }
     }
 
     class Groups implements StorageStatement.Groups
     {
-
         @Override
         public PageCursor openPageCursor( long reference )
         {
-            return neoStores.getRelationshipGroupStore().openPageCursorForReading( reference );
+            return relationshipGroupStore.openPageCursorForReading( reference );
         }
 
         @Override
         public void loadRecordByCursor( long reference, RelationshipGroupRecord relationshipGroupRecord,
                 RecordLoad mode, PageCursor cursor ) throws InvalidRecordException
         {
-            neoStores.getRelationshipGroupStore().getRecordByCursor( reference, relationshipGroupRecord, mode, cursor );
+            relationshipGroupStore.getRecordByCursor( reference, relationshipGroupRecord, mode, cursor );
         }
 
         @Override
         public long getHighestPossibleIdInUse()
         {
-            return neoStores.getRelationshipGroupStore().getHighestPossibleIdInUse();
+            return relationshipGroupStore.getHighestPossibleIdInUse();
         }
     }
 
     class Properties implements StorageStatement.Properties
     {
-
         @Override
         public PageCursor openPageCursor( long reference )
         {
-            return neoStores.getPropertyStore().openPageCursorForReading( reference );
+            return propertyStore.openPageCursorForReading( reference );
         }
 
         @Override
         public void loadRecordByCursor( long reference, PropertyRecord propertyBlocks, RecordLoad mode,
                 PageCursor cursor ) throws InvalidRecordException
         {
-            neoStores.getPropertyStore().getRecordByCursor( reference, propertyBlocks, mode, cursor );
+            propertyStore.getRecordByCursor( reference, propertyBlocks, mode, cursor );
         }
 
         @Override
         public long getHighestPossibleIdInUse()
         {
-            return neoStores.getPropertyStore().getHighestPossibleIdInUse();
+            return propertyStore.getHighestPossibleIdInUse();
         }
 
         @Override
         public PageCursor openStringPageCursor( long reference )
         {
-            return neoStores.getPropertyStore().getStringStore().openPageCursorForReading( reference );
+            return propertyStore.getStringStore().openPageCursorForReading( reference );
         }
 
         @Override
         public PageCursor openArrayPageCursor( long reference )
         {
-            return neoStores.getPropertyStore().getArrayStore().openPageCursorForReading( reference );
+            return propertyStore.getArrayStore().openPageCursorForReading( reference );
         }
 
         @Override
         public ByteBuffer loadString( long reference, ByteBuffer buffer, PageCursor page )
         {
-            return readDynamic( neoStores.getPropertyStore().getStringStore(), reference, buffer, page );
+            return readDynamic( propertyStore.getStringStore(), reference, buffer, page );
```

Comment:
It'd sort of make sense to keep `propertyStore.getStringStore()` as a field too.